src/patternfly/components/OptionsMenu/docs/code.md

```diff
@@ -24,5 +24,8 @@
 | `.pf-m-top` | `.pf-c-options-menu` | Modifies to display the menu above the toggle. |
 | `.pf-m-align-right` | `.pf-c-options-menu__menu` | Modifies to display the menu aligned to the right edge of the toggle |
 | `.pf-m-expanded` | `.pf-c-options-menu` |  Modifies for the expanded state. |
+| `.pf-m-hover` | `.pf-c-options-menu__toggle`, `.pf-c-options-menu__toggle .pf-m-plain` | Modifies for the hover state. |
```

Comment:
do you need `.pf-c-options-menu__toggle .pf-m-plain` here? Seems like that's a typo, too (should be ``.pf-c-options-menu__toggle.pf-m-plain`) but it seems like listing `.pf-c-options-menu__toggle` covers it.

Reply:
Agreed.